src/codemap_details/Components/CodeMapBanner.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import styles from "../CSS/CodeMapBanner.module.css";
```

Comment:
Change everywhere from CSS to Css

Reply:
@SinghVikram97 As @vinitshahdeo suggested, lets have the folder name as CSS rather than Css, 

@harsha14gupta Change the folder name, and let the import be same.